bin/stencil-start

```diff
@@ -1,34 +1,26 @@
 #!/usr/bin/env node
 
-var Bs = require('browser-sync').create();
-var Cycles = require('../lib/cycles');
-var recursiveRead = require('recursive-readdir');
-var Async = require('async');
-var templateAssembler = require('../lib/template-assembler');
-var Fs = require('fs');
-var JspmAssembler = require('../lib/jspm-assembler');
-var Path = require('path');
-var Pkg = require('../package.json');
-var Program = require('commander');
-var Server = require('../server');
-var ThemeConfig = require('../lib/theme-config');
-var buildConfig = require('../lib/build-config');
-var Url = require('url');
-var Wreck = require('wreck');
-var jsonLint = require('../lib/json-lint');
-var versionCheck = require('../lib/version-check');
-var themePath = process.cwd();
-var browserSyncPort;
-var templatePath = Path.join(themePath, 'templates');
-var dotStencilFilePath = Path.join(themePath, '.stencil');
-var themeConfigPath = Path.join(themePath, 'config.json');
-var dotStencilFile;
-var stencilServerPort;
-var themeConfig;
-var configuration;
-var staplerUrl;
-var headers;
-let tunnel;
+const Bs = require('browser-sync').create();
+const Cycles = require('../lib/cycles');
+const recursiveRead = require('recursive-readdir');
+const Async = require('async');
+const templateAssembler = require('../lib/template-assembler');
+const Fs = require('fs');
+const JspmAssembler = require('../lib/jspm-assembler');
+const Path = require('path');
+const Pkg = require('../package.json');
+const Program = require('commander');
+const Server = require('../server');
+const ThemeConfig = require('../lib/theme-config');
+const buildConfig = require('../lib/build-config');
+const Url = require('url');
+const Wreck = require('wreck');
+const jsonLint = require('../lib/json-lint');
+const versionCheck = require('../lib/version-check');
+let themePath = process.cwd();
+let templatePath = Path.join(themePath, 'templates');
+let dotStencilFilePath = Path.join(themePath, '.stencil');
+let themeConfigPath = Path.join(themePath, 'config.json');
```

Comment:
Is there a reason why these paths can't be const ?

Reply:
no, I just didn’t notice that they aren’t further modified

Reply:
updated